vspace/src/main/webapp/WEB-INF/views/staff/spaces/spacelist.html

```diff
@@ -1,185 +1,185 @@
+	                        aria-hidden="true"
+	                        style="color: var(--secondary); padding-right: 8px;"></i>
+	                    <span th:if="${!space.hasIncomingLinks()}"
+	                        class="noIncomingLinkMessage">This space
+	                        has no incoming links.</span> <a
+	                        th:onclick="javascript:checkSpaceLinkPresent('[(${space.id})]', '[(@{'/staff/'})]', '[('?'+${_csrf.parameterName}+'='+${_csrf.token})]',$('#headerSpaceValue'))"
+	                        class="checkSpaceLinkPresent"
+	                        data-toggle="modal"> <span
+	                        class="checkSpaceLinkPresent" id="deleteSpace"
+	                        style="color: #666;"><i class="fas fa-trash-alt" style="color: var(--error)"></i></span>
+	                    </a>
+	                </div>
+	            </li>
+	        </ul>
+        </div>
+        
+        <div class="modal fade" id="confirm-space-delete" tabindex="-1"
+            role="dialog" aria-labelledby="myModalLabel"
+            aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header" data-spaceValue=""
+                        id="headerSpaceValue">
+                        <h5 class="modal-title" id="deleteModalTitle">
+                            Confirm Deletion?</h5>
+                        <button type="button" class="close"
+                            data-dismiss="modal" aria-hidden="true">x</button>
+                    </div>
+                    <div class="modal-body">
+                        <p id="spaceData">Are you sure you want to
+                            delete ?</p>
+                        <div id="warningMessage">
+                            <div class="text-danger">Other spaces
+                                have links to this space!</div>
+                        </div>
+                        <div id="exhibitionMessage">
+                            <div class="text-danger">This space is
+                                the start of the exhibition. Deleting it
+                                will make your exhibition unavailable.</div>
+                        </div>
+                        <div id="finalWarning">
+                            <div class="text-danger">Do you want
+                                to continue?</div>
+                        </div>
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" id="closeButton"
+                            class="btn btn-default" data-dismiss="modal">Cancel</button>
+                        <button type="button"
+                            class="btn btn-danger btn-ok">Yes,
+                            delete!</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
```

Comment:
what has been changed in this file? why does it show up as all changed?

Reply:
I fixed the line endings, because I am not sure what files endings must have changed when I merge `VSPC-120`. So I changed line endings of all the html files.

Reply:
That's what I thought, but when I downloaded the old version, it had the correct line endings.